.rhcicd/pr_check.sh

```diff
@@ -31,8 +31,13 @@ source $CICD_ROOT/cji_smoke_test.sh
 
 # Until test results produce a junit XML file, create a dummy result file so Jenkins will pass
 mkdir -p $WORKSPACE/artifacts
-cat << EOF > ${WORKSPACE}/artifacts/junit-dummy.xml
-<testsuite tests="1">
-    <testcase classname="dummy" name="dummytest"/>
-</testsuite>
-EOF
+
+# adding floorist tests.
+export COMPONENT_NAME="policies-engine"
+# I also need to pass in FLOORPLAN_2_TEST.
+export IQE_ENV_VARS="FLOORPLAN_TO_TEST=policies-backend-hms"
```

Comment:
Would we support more than one floorplan to test? For example by providing more plan names separated by comma?

Reply:
we could support multiple floorplans. However, I'd like to do this in another story/task for the floorist plugin. It should not be a huge effort. I actually thought about implementing this initially but we did not state we needed it.

Reply:
I think we've discussed this as one of the requirements, as services already use two floorplans. Anyways, we can extend it later.

Reply:
I added support for multiple floorplans: https://gitlab.cee.redhat.com/insights-qe/iqe-floorist-plugin/-/merge_requests/24